Sources/ShowAttachments.php

```diff
@@ -185,7 +181,7 @@ function showAttachment()
 	if (!file_exists($file['filePath']))
 	{
 		header((preg_match('~HTTP/1\.[01]~i', $_SERVER['SERVER_PROTOCOL']) ? $_SERVER['SERVER_PROTOCOL'] : 'HTTP/1.0') . ' 404 Not Found');
```

Comment:
It used to be far fewer but it seems that 2.1 has had lots of shenanigans and new routes through the application than I remember it having.